util/unittests/test_html.py

```diff
@@ -202,7 +202,7 @@ def test01_multy_dict_to_html(self):
                              ('Testing multy_dict_to_'
                               'html_multiline output test 2'))
 
-    def test02_multy_dict_to_html(self):
+    def test_multy_dict_to_html(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_multy_dict_to_html' from line 192

<!-- Comment by [SideCI](https://sideci.com) -->